com.officefactoring.officefactoring.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: com.officefactoring.officefactoring
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+rename-icon: officefactoring
+command: OfficeFactoring
+modules:
+  - name: OfficeFactoring
+    buildsystem: simple
+    build-commands:
+      - install -D OfficeFactoring /app/bin/OfficeFactoring
```

Comment:
let me guess this is an x86_64 binary and you still have flatpak build an aarch64 package.